osgp/protocol-adapter-iec60870/README.md

```diff
@@ -10,3 +10,64 @@ Grid eXchange Fabric detailed documentation:
 
 Grid eXchange Fabric issue tracker:
 * [Grid eXchange Fabric Jira](https://smartsocietyservices.atlassian.net/projects/OC/issues/)
+
+## Add 104 devices to the database
+
+...
+-- osgp_core
+
+INSERT INTO device
+    (creation_time, modification_time, version, device_identification, device_type, is_activated,
+     protocol_info_id, network_address, in_maintenance, gateway_device_id,
+     device_model, device_lifecycle_status)
+  VALUES
+    (current_timestamp, current_timestamp, 0, 'GATEWAY_1', null, true,
+        (SELECT id FROM protocol_info WHERE protocol = '60870-5-104' AND protocol_version = '1.0'),
+        '127.0.0.1', false, null,
+        (SELECT id FROM device_model WHERE model_code = 'Test' AND manufacturer_id = (SELECT id FROM manufacturer WHERE code = 'Test')),
+        'IN_USE');
+INSERT INTO device
+    (creation_time, modification_time, version, device_identification, device_type, is_activated,
+     protocol_info_id, network_address, in_maintenance, gateway_device_id,
+     device_model, device_lifecycle_status)
+  VALUES
+    (current_timestamp, current_timestamp, 0, 'LMD_1', 'LMD', true,
+        (SELECT id FROM protocol_info WHERE protocol = '60870-5-104' AND protocol_version = '1.0'),
+        '127.0.0.1', false, (SELECT id FROM device WHERE device_identification = 'GATEWAY_1'),
+        (SELECT id FROM device_model WHERE model_code = 'Test' AND manufacturer_id = (SELECT id FROM manufacturer WHERE code = 'Test')),
+        'IN_USE'),
+    (current_timestamp, current_timestamp, 0, 'LMD_2', 'LMD', true,
+        (SELECT id FROM protocol_info WHERE protocol = '60870-5-104' AND protocol_version = '1.0'),
+        '127.0.0.1', false, (SELECT id FROM device WHERE device_identification = 'GATEWAY_1'),
+        (SELECT id FROM device_model WHERE model_code = 'Test' AND manufacturer_id = (SELECT id FROM manufacturer WHERE code = 'Test')),
+        'IN_USE');
+
+INSERT INTO rtu_device VALUES ((SELECT id FROM device WHERE device_identification = 'GATEWAY_1'), null, 2);
+
+INSERT INTO light_measurement_device
+    (id, description, code, color, digital_input)
+  VALUES
+    ((SELECT id FROM device WHERE device_identification = 'LMD_1'), 'Test LMD 1', 'CODE-1', '#c9eec9', 42),
```

Comment:
We probably don't need to set the digital_input for the IEC-60870 light measurement devices as the protocol adapter will have the information for mapping the IOAs to a specific light measurement device.